Automattic/wp-calypso
# Checklist: Translate the onboarding checklist items

Since we decided to roll out the onboarding checklist to 100% users, the checklist items should be translated before #25929 is merged.

The duration has now become "minute" or "minutes".

## Screens

![jp](https://user-images.githubusercontent.com/841763/42649031-ceeb285e-8608-11e8-8f2e-1b68131ac25b.png)
![minutes](https://user-images.githubusercontent.com/841763/42649032-cf147ede-8608-11e8-9488-f51c4684d9a7.png)


# How to test

1. Create a free website with a non-English user account.
2. You should be able to see the checklist items translated in your language.
